.circleci/create_schema.cql

```diff
@@ -0,0 +1,80 @@
+DROP KEYSPACE IF EXISTS scalar;
```

Comment:
Since it is a public repository, I would like this schema information to be not disclosed.
I did a quick investigation and there might be a way.
https://support.circleci.com/hc/en-us/articles/360006717953-Storing-secret-files-certs-etc-

Reply:
Okay, how about i use environment variable and the file content won't be visible to the public...

Reply:
That could also be a way. You mean storing all the string in the env variable, right ?

Reply:
Yes

Reply:
OK, let's try the simpler approach !